### ISSUE TITLE
Potential typo in documentation

### ISSUE THREAD

**Issue by J-Moravec** [gbm-developers/gbm#69]
There might be a typo in documentation in the following lines of the `gbm()` function:

https://github.com/gbm-developers/gbm/blob/18e595597e19c289328f5eb0b73945a1ea72fc7d/R/gbm.R#L25-L32

```
#' Currently available options are \code{"gaussian"} (squared error), 
#' \code{"laplace"} (absolute loss), \code{"tdist"} (t-distribution loss), 
#' \code{"bernoulli"} (logistic regression for 0-1 outcomes), 
#' \code{"huberized"} (huberized hinge loss for 0-1 outcomes), classes),
```

The last bracket does not have a matching bracket, and the word "classes" doesn't fit in the context, I presume that it relates to something else? Possibly a deleted mention of multinomial error, which is mentioned in the previous section of the documentation, but is missing here?

**Comment by bgreenwell** [comment]
Thanks @J-Moravec!

**Comment by gregridgeway** [comment]
Fixed with version 2.1.9
